app/api/diagnostic/route.ts

```
@@ -485,7 +462,7 @@
         }
         
         // Simplified recommendations
-        let recommendations = ["Focus on areas where you were unsure."];
+        const recommendations = ["Focus on areas where you were unsure."];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)